lib/types/api.types.ts

```diff
@@ -271,29 +271,56 @@ export type DialogExtensionSDK = Omit<BaseExtensionSDK, 'ids'> & {
   window: WindowAPI
 }
 
-export type PageExtensionSDK = Omit<BaseExtensionSDK, 'ids'> & {
+export type PageAppSDK = Omit<BaseAppSDK, 'ids'> & {
   /** A set of IDs actual for the app */
   ids: Omit<IdsAPI, EntryScopedIds>
 }
 
-export type HomeExtensionSDK = Omit<BaseExtensionSDK, 'ids'> & {
+export type HomeAppSDK = Omit<BaseAppSDK, 'ids'> & {
   ids: Omit<IdsAPI, EntryScopedIds>
 }
 
-export type AppExtensionSDK = Omit<BaseExtensionSDK, 'ids'> & {
+export type ConfigAppSDK = Omit<BaseAppSDK, 'ids'> & {
   /** A set of IDs actual for the app */
   ids: Omit<IdsAPI, EntryScopedIds | 'extension' | 'app'> & { app: string }
   app: AppConfigAPI
 }
 
-export type KnownSDK =
-  | FieldExtensionSDK
-  | SidebarExtensionSDK
-  | DialogExtensionSDK
-  | EditorExtensionSDK
-  | PageExtensionSDK
-  | AppExtensionSDK
-  | HomeExtensionSDK
+export type KnownAppSDK =
+  | FieldAppSDK
+  | SidebarAppSDK
+  | DialogAppSDK
+  | EditorAppSDK
+  | PageAppSDK
+  | ConfigAppSDK
+  | HomeAppSDK
+
+/** @deprecated consider using {@link BaseAppSDK} */
+export type BaseExtensionSDK = BaseAppSDK
+
+/** @deprecated consider using {@link EditorAppSDK} */
+export type EditorExtensionSDK = EditorAppSDK
+
+/** @deprecated consider using {@link SidebarAppSDK} */
+export type SidebarExtensionSDK = SidebarAppSDK
+
+/** @deprecated consider using {@link FieldAppSDK} */
+export type FieldExtensionSDK = FieldAppSDK
+
+/** @deprecated consider using {@link DialogAppSDK} */
+export type DialogExtensionSDK = DialogAppSDK
+
+/** @deprecated consider using {@link PageAppSDK} */
+export type PageExtensionSDK = PageAppSDK
+
+/** @deprecated consider using {@link HomeAppSDK} */
+export type HomeExtensionSDK = HomeAppSDK
+
+/** @deprecated consider using {@link ConfigAppSDK} */
+export type AppExtensionSDK = ConfigAppSDK
+
+/** @deprecated consider using {@link KnownAppSDK} */
+export type KnownSDK = KnownAppSDK
```

Comment:
`KnownSDK` is still used in `api.ts`, `index.ts` and `initialize.ts`

Reply:
Good catch 👍